plexus/templates/main/alias_detail.html

```diff
@@ -66,14 +66,23 @@ <h3 id="myModalLabel1">Request Alias Change for {{object.hostname}}</h3>
 		above instead.</p>
 			
 		<label>New Server
-			<select name="new_server">
+			<select name="new_ipaddress">
 				{% for server in all_servers %}
-				{% if server.ipaddress_set.count %}
-				{% ifequal server.id object.ip_address.server.id %}
-				{% else %}
-				<option value="{{server.id}}">{{server.name}}</option>
-				{% endifequal %}
-				{% endif %}
+				  {% ifequal server.ipaddress_set.count 1 %}
```

Comment:
ifequal is no longer necessary because we can do

    {% if server.ipaddress_set.count == 1 %}

https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#ifequal-and-ifnotequal